src/lib/protocols/bittorrent.c

```diff
@@ -68,7 +76,13 @@ static u_int8_t is_utpv1_pkt(const u_int8_t *payload, u_int payload_len) {
   if(h->h_version != 1)             return(0);
   if(h->h_type > 4)                 return(0);
   if(h->next_extension > 2)         return(0);
-  if(ntohl(h->window_size) > 65565) return(0);
```

Comment:
Checking window size is not useful as different implementations enforce different window size's.